src/main/java/edu/wright/airviewer2/AIRViewerModel.java

```diff
@@ -27,9 +27,13 @@ public class AIRViewerModel extends DocumentCommandWrapper {
     //private static final Logger logger = Logger.getLogger(AIRViewerModel.class.getName());
 
```

Comment:
please remove the logger since it is not used at all.

Reply:
It is done